app.js

```diff
@@ -56,7 +56,7 @@ app.post('/login', routes.loginHandler);
 app.get('/admin', routes.isLoggedIn, routes.admin);
 app.get('/account_details', routes.isLoggedIn, routes.get_account_details);
 app.post('/account_details', routes.isLoggedIn, routes.save_account_details);
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; NoSQL Injection

Unsanitized input from `the HTTP request body` flows into `find`, where it is used in an NoSQL query. This may result in an NoSQL Injection vulnerability.

Line 81 | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 555</a> | <a href="https://cwe.mitre.org/data/definitions/319.html">CWE-319</a> 

<details>

<summary>Data flow: 3 steps</summary>

https://github.com/trailblazing-dx/tiago_pr_playground/blob/855a1fa7877e00fd6c9f4c15ef769d8f5ea3b0cf/app.js#L52

https://github.com/trailblazing-dx/tiago_pr_playground/blob/855a1fa7877e00fd6c9f4c15ef769d8f5ea3b0cf/app.js#L75

https://github.com/trailblazing-dx/tiago_pr_playground/blob/855a1fa7877e00fd6c9f4c15ef769d8f5ea3b0cf/app.js#L80

</details>